src/tests/test_magic_cmd.py

```diff
@@ -70,3 +70,42 @@ def test_columns_with_schema(ip, tmp_empty):
     ).result._repr_html_()
 
     assert "some_number" in out
+
+
+def test_sql_cmd_magic_lessthan(ip):
```

Comment:
create a new table with some sample (i.e., use `INSERT INTO`) and add tests bor both cases: then the data fulfills the conditions and when it doesn't